layers/+emacs/org/README.org

```diff
@@ -155,6 +157,46 @@ Alternatively, add the following line to each =.org= file you want to process:
   #+REVEAL_ROOT: https://cdn.jsdelivr.net/npm/reveal.js@3.8.0
 #+END_EXAMPLE
 
+** Org-contacts supported
+[[https://github.com/tkf/org-mode/blob/master/contrib/lisp/org-contacts.el][org-contacts]] is a simple contacts managemant system.
+
+To install org-contacts, set the variable =org-enable-org-contacts-support= to
+=t=. Optionally, also set the variable =org-contacts-files= and add a capture
+template. The value of the =org-contacts-files= variable should be a list with
+filenames to use as contact sources. If set to =nil= (default) then all you Org
+files will be used. The first file in the =org-contacts-files= list can be
+visited with the keyboard shortcut ~SPC a o C f~.
+#+BEGIN_SRC emacs-lisp
+  (setq-default dotspacemacs-configuration-layers
+                '((org :variables org-enable-org-contacts-support t
+                       org=contact-file '("file1.org" "file2.org")
+                       ("c" "Contacts" entry (file "~/Org/contacts.org")
+                        "* %(org-contacts-template-name)
+  :PROPERTIES:
+  :EMAIL: %(org-contacts-template-email)
+  :END:")))
+                )))
+#+END_SRC
+A more elaborate capture template can be found in the =org-contacts.el= file.
+
+Contacts can include the :BIRTHDAY: keyword. To include the birthdays in your
+org-agenda add ~%%(org-contacts-anniversaries)~ to one of your contacts files.
+It is important that this is inserted after a heading an that it has no
+preceding whitespace. So probably the best way is to add
+#+begin_src emacs-lisp :tangle yes
+* Birthdays
+%%(org-contacts-anniversaries)
+#+end_src
+to the beginning or the end of one of your contacts files.
+
+The [[https://github.com/flexibeast/org-vcard][org-vcard]] package may be used to import/export vcards. This package can be
```

Comment:
When you mention `org-vcard` I think it would be more useful to directly integrate it into the layer basically you have already with the instructions below ;)